WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -234,6 +234,7 @@ class WordPressAppDelegate: UIResponder, UIApplicationDelegate {
 
         // Deferred tasks to speed up app launch
         DispatchQueue.global(qos: .background).async { [weak self] in
+            AccountService(managedObjectContext: ContextManager.shared.mainContext).mergeDuplicatesIfNecessary()
```

Comment:
This is called on a background queue with a main context, it should be wrapped in a `context.performBlock()`

Reply:
Good point. To keep things tidy I've moved this into a separate method to wrangle the perform block.